pyflagser/tests/test_flagser.py

```diff
@@ -227,6 +228,19 @@ def test_filtrations_d5(flag_file, filtration):
                 .format(tmp, tmp2)
 
 
+def compute_weighted(adjacency_matrix):
```

Comment:
Why do we need this? It seems to just be a renaming of `flagser_unweighted`

Reply:
Because my brain sometimes misses the point ...
We don't need this function, and even the function name is wrong, god dammit
Let me fix this